ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/ConsumerGroupsTable.stories.tsx

```diff
@@ -12,1753 +12,67 @@ export default {
-            },
-            {
-              lag: 400,
-            },
-          ],
-          members: [
-            {
-              host: "localhost",
-              memberId: "member-1",
-              clientId: "client-1",
-              groupInstanceId: "instance-1",
-              assignments: [
-                {
-                  topicName: "console_datagen_002-a",
-                  topicId: "1",
-                },
-                {
-                  topicName: "console_datagen_002-b",
-                  topicId: "2",
-                },
-              ],
-            },
-          ],
-        },
-      },
-      {
-        id: "console-datagen-group-2",
-        attributes: {
-          state: "EMPTY",
-          offsets: [
-            {
-              lag: 200,
-            },
-            {
-              lag: 400,
-            },
-            {
-              lag: 400,
-            },
-          ],
-          members: [
-            {
-              host: "localhost",
-              memberId: "member-1",
-              clientId: "client-1",
-              groupInstanceId: "instance-1",
-              assignments: [
-                {
-                  topicName: "console_datagen_002-a",
-                  topicId: "1",
-                },
-                {
-                  topicName: "console_datagen_002-b",
-                  topicId: "2",
-                },
-              ],
-            },
-          ],
-        },
+const generateConsumerGroup = (
+  id: string,
+  state: string,
+  lag1: number,
+  lag2: number,
+  lag3?: number,
+) => ({
+  id,
+  attributes: {
+    state,
+    offsets:
+      lag3 !== undefined
+        ? [{ lag: lag1 }, { lag: lag2 }, { lag: lag3 }]
+        : [{ lag: lag1 }, { lag: lag2 }],
+    members: [
+      {
+        host: "localhost",
+        memberId: "member-1",
+        clientId: "client-1",
+        groupInstanceId: "instance-1",
+        assignments: [
+          {
+            topicName: `console_datagen_${id.split("-").pop()}-a`,
+            topicId: "1",
+          },
+          {
+            topicName: `console_datagen_${id.split("-").pop()}-b`,
+            topicId: "2",
+          },
+        ],
       },
     ],
   },
+});
+
+export const ConsumerGroups: Story = {
+  args: {
+    perPage: 20,
+    consumerGroups: (() => {
+      const groups = Array.from({ length: 21 }, (_, i) =>
+        generateConsumerGroup(
+          `console-datagen-group-${i}`,
+          ["STABLE", "EMPTY"][i % 2],
+          i * 1000,
+          i * 1000,
+          i % 3 === 0 ? i * 1000 : undefined,
+        ),
+      );
+      return groups;
+    })(),
+    total: (() => {
+      const groups = Array.from({ length: 21 }, (_, i) =>
+        generateConsumerGroup(
+          `console-datagen-group-${i}`,
+          ["STABLE", "EMPTY"][i % 2],
+          i * 1000,
+          i * 1000,
+          i % 3 === 0 ? i * 1000 : undefined,
+        ),
+      );
+      return groups.length;
+    })(),
```

Comment:
One question actually on this part - is there any different between the lambda and simply `total: 21` ?

Reply:
Yes, we can simply write the total number instead of using a lambda function, as the array creation is unnecessary. I just did it to match the number of consumers.

Reply:
No worries, just confirming my understanding of it. I guess the number makes it most obvious, but I'm fine with either.